pandas/tests/extension/base/interface.py

```diff
@@ -29,6 +29,24 @@ def test_can_hold_na_valid(self, data):
         # GH-20761
         assert data._can_hold_na is True
 
+    def test_contains(self, data):
+        # GH-37867
+        scalar = data[~data.isna()][0]
+
+        assert scalar in data
+
+        na_value = data.dtype.na_value
+
+        if data.isna().any():
+            assert na_value in data
```

Comment:
`data` is not guaranteed to have a missing values, but there is also a `data_missing` fixture that has this guarantee and which you can add here to the test, and use that here. 
And then to be sure we test the case of data without NAs, maybe do the else with `data[~data.isna()]` (that is guaranteed to not be empty)

Reply:
And thanks for this base test!

Reply:
Ok, I've added a test method using data_missing.

Reply:
BTW, you can pass both fixtures to the same test, then you can write the test with less duplication as it is now

Reply:
you can pass them as the string name of the test and use: `frame = request.getfixturevalue(fixture_func_name)` to create the actual frame